python/cudf/cudf/__init__.py

```diff
@@ -34,3 +34,15 @@
 
 __version__ = get_versions()["version"]
 del get_versions
+
+
+# Import dask_cudf dispatch functions
+# (In case they are not correctly imported in distributed)
+try:
+    from dask_cudf.backends import (
+        hash_object_cudf,
+        hash_object_cudf_index,
+        group_split_cudf,
+    )
+except ImportError:
+    pass
```

Comment:
Do we need these top level imports? I thought it was a dask bug that has been resolved?

Reply:
It is resolved.  I just removed these imports - Thanks!